### DISCUSSION TITLE
Delete `0`

### DISCUSSION THREAD

**Discussion by 25Das** [prettier/prettier#16161] [Q&A] [answered]
I am using prettier + eslint and it keeps deleting zeros from double digits float numbers ending in 0 (10000.00 -> 10000.0).
This totally breaks frontend input masks.
How to disable this on configuration? I dont want to add a million disable eslint comments through my codebase nor do I want to parse a million different numbers everytime they are used because prettier finds it unecessary and I cant find a rule to disable.

**Comment by kachkaev** [comment, marked as answer]
👋  @25Das! The suggested way of integrating Prettier and ESLint is via https://www.npmjs.com/package/eslint-plugin-prettier. This essentially turns off _all_ ESLint rules that may conflict with Prettier output.

If you do need to format decimals somewhere (e.g. in a list of test cases or in seed data), adding `// prettier-ignore` is the way to go:

```ts
// prettier-ignore
const myValue = 10000.00;
```
[Playground](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEB6VACADgJzjGASzhwFpCBzKCPAHSkigGcYMBbATwDUBDAGwCucDAF4MARgAM0yQDppAbnr1GLdt35CATKIky5ildDWdeguAGZdUmfMlKo9dNjwFiZStToNjrU5rgAFmt9OwcQABoQCCwiY2RQHhwcCAB3AAUkhCZkEH5Ung4cqIAjHB4wAGt8AGUeNjgAGUIoOGQAM34mOFLyqtqsCpaKZBgcISjutkJR8Z6QOAAPLBJCBth+ABUSKCTiHKROvm7J4b44AEUBCHgOrvmAKyZFmrPL69vD+6iARw+4dIpLAHPJMUitOAAEyhkRAYx4hD4wwAwhA2GweLl+HxYUw3gBBAg4QglATwdIkZqtO7HeYACxgbD4AHU6YR4ExBmA4DVsuzCAA3dkcXJgJjFEACoQASSg0NgNTAxNi+LlNRgHHONJOIFwEG6zPKWFyuDg3RwAraURa5pggJ4FAx2vmgxw5tyJR4JTgOKiuBaMGZhEhMDpyAAHJIong-oQ8PbHZivrSojAvUGQ2GkFoogJupsvQcjjq4GxvZDoZDGjwoBQBA64AAxGgYty1rFkiAgAC+3aAA)

If you want to keep double digits in the entire codebase, Prettier might not be the right option for you (unless you want to fork it). See also: https://github.com/prettier/prettier/issues/5287
